plugins/NebulaLogger/classes/RollupNebulaLoggerAdapter.cls

```diff
@@ -1,4 +1,14 @@
 public class RollupNebulaLoggerAdapter extends RollupLogger {
+  public RollupNebulaLoggerAdapter() {
```

Comment:
Nice! I like this code you've added in the constructor

Reply:
Thank you so much for working with me to get this functionality supported! I created a few logs while testing these changes and the updated Origin Location and stacktraces are so much cleaner as a result.

Reply:
Sweet! Yeah, I really like the new `Logger.ignoreOrigin()`, it seems like a lot of developers have a use case for it